front/src/styles/scss/applications/operationalStudies/_managetrainschedule.scss

```diff
@@ -42,6 +42,16 @@
   }
 }
 
+.flex-on-line {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.flex-on-line .select-improved {
+  flex-grow:4
+}
+
```

Comment:
Should be in dedicated scss linked to select-improved, not here

Reply:
ok moved